pymysqlreplication/tests/test_basic.py

```diff
@@ -1473,6 +1472,41 @@ def test_rows_query_log_event(self):
         self.assertIsInstance(event, RowsQueryLogEvent)
 
 
+class TestGtidEvent(base.PyMySQLReplicationTestCase):
+    def setUp(self):
+        super(TestGtidEvent, self).setUp()
+        self.execute("SET SESSION binlog_rows_query_log_events=1")
+
+    def tearDown(self):
+        self.execute("SET SESSION binlog_rows_query_log_events=0")
+        super(TestGtidEvent, self).tearDown()
+
+    def test_gtid_event(self):
+        self.stream.close()
+        self.stream = BinLogStreamReader(
+            self.database,
+            server_id=1024,
+            only_events=[GtidEvent],
+        )
```

Comment:
@Bue-von-hon 
may be we did not pass FORMAT_DESCRIPTION_EVENT
so mysql_version is not initalized.
Could you change Like this?
```suggestion
        self.stream = BinLogStreamReader(
            self.database,
            server_id=1024,
            only_events=[GtidEvent, FORMAT_DESCRIPTION_EVENT],
        )
```